tests/Control/SMIMEMailerTest.php

```diff
@@ -0,0 +1,243 @@
+<?php
+
+namespace SilverStripe\SMIME\Tests\Control;
+
+use Exception;
+use PHPUnit\Framework\MockObject\MockObject;
+use SilverStripe\Control\Email\Email;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\SMIME\Control\SMIMEMailer;
+use Swift_ByteStream_FileByteStream;
+
+class SMIMEMailerTest extends SapphireTest
+{
+
+    protected $usesDatabase = false; // phpcs:ignore
+
+    public function testEncryptionOnly()
+    {
+        $recipientEncryptionCrt = $this->getAsset('/assets/smime_test_recipient.crt');
+        $email = $this->buildEmail('Email with encryption', 'This is a confidential email.');
+        $partialMockMailer = $this->createPartialMockedMIMEMailer([$recipientEncryptionCrt]);
+
+        $partialMockMailer->expects($this->once())->method('sendSwift')->with(
+            self::callback(function ($message) {
+                $encryptedMessage = $message->toString();
+                self::assertStringContainsString('Content-Type: application/x-pkcs7-mime;', $encryptedMessage);
+                self::assertStringNotContainsString('This is a confidential email.', $encryptedMessage);
+
+                return true;
+            })
+        );
```

Comment:
It is good to try and relate the code wherever possible.

Maybe worth adding a new variable `$unencryptedContent` to represent the content we use and shouldn't see in the encrypted message.

```suggestion
        $unencryptedContent = 'This is a confidential email.';
        $email = $this->buildEmail('Email with encryption', $unencryptedContent);
        $partialMockMailer = $this->createPartialMockedMIMEMailer([$recipientEncryptionCrt]);

        $partialMockMailer->expects($this->once())->method('sendSwift')->with(
            self::callback(function ($message) {
                $encryptedMessage = $message->toString();
                self::assertStringContainsString('Content-Type: application/x-pkcs7-mime;', $encryptedMessage);
                self::assertStringNotContainsString($unencryptedContent, $encryptedMessage);

                return true;
            })
        );
```

Reply:
Ah, I originally tried to do this but had some scope issues, but now realise I just need to pass in the variable to the function call as a use statement

Reply:
ahh, sorry...didn't spot that, nice work for fixing 👍